apps/coscrad-frontend/src/store/slices/resources/terms/thunks/fetch-terms.ts

```diff
@@ -1,33 +1,47 @@
-import { IIndexQueryResult, ITermViewModel, WithTags } from '@coscrad/api-interfaces';
+import { CategorizableType, IIndexQueryResult, ITermViewModel } from '@coscrad/api-interfaces';
 import { isNullOrUndefined } from '@coscrad/validation-constraints';
 import { getConfig } from '../../../../../config';
+import { getConfigurableContent } from '../../../../../configurable-front-matter';
 import { buildResourceFetchActionPrefix } from '../../../utils/build-resource-fetch-action-prefix';
 import { createFetchThunk } from '../../../utils/create-fetch-thunk';
 import { getApiResourcesBaseRoute } from '../../shared';
 import { TERMS } from '../constants';
-import { TermIndexState } from '../types/term-index-state';
 
-export const fetchTerms = createFetchThunk<TermIndexState>(
+export const fetchTerms = createFetchThunk<IIndexQueryResult<ITermViewModel>>(
     buildResourceFetchActionPrefix(TERMS),
     `${getApiResourcesBaseRoute()}/terms`,
-    (
-        serverResponse: IIndexQueryResult<WithTags<ITermViewModel>>
-    ): IIndexQueryResult<WithTags<ITermViewModel>> => {
+    (serverResponse: IIndexQueryResult<ITermViewModel>): IIndexQueryResult<ITermViewModel> => {
         const { apiUrl } = getConfig();
 
+        const { indexToDetailFlows } = getConfigurableContent();
+
+        const termIndexToDetailFlowConfig = indexToDetailFlows.find(
+            ({ categorizableType }) => categorizableType === CategorizableType.term
+        );
+
+        const identityFilter = (x: unknown) => x;
```

Comment:
What does this do?  Just curious

Reply:
Nothing. I was hoping I made it obvious.

Reply:
This is probably passed into a `filter` or `map` somewhere.